examples/platform/set_user_enable.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020-2021. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""Set a user's enable/disable status from the command line."""
+
+from cbc_sdk.helpers import build_cli_parser, get_cb_cloud_object
+from cbc_sdk.platform import User
+import sys
+
+
+def get_status(user):
+    """Returns True if user is enabled, False if disabled."""
+    grant = user.grant()
+    return not grant.revoked
+
+
+def update_status(user, value):
+    """Sets the enable/disable status of a user."""
+    grant = user.grant()
+    for profile in grant.profiles_:
+        profile.set_disabled(not value)
+    grant.revoked = not value
+    grant.save()
+
+
+def main():
+    """Main function for the role changing script"""
+    parser = build_cli_parser("Set User Enable/Disable")
+    parser.add_argument('email', help='E-mail address of the user to set status of')
+    parser.add_argument('-e', '--enable', action='store_true', help='Enable this user')
+    parser.add_argument('-d', '--disable', action='store_true', help='Disable this user')
```

Comment:
The only thing I was thinking is whether there should be some validations that -e and -d are not both present, but considering that only enabled would be applied maybe we do not need to worry.

Reply:
Probably not. This is just an example and everyone can see the control flow favors "enable" over "disable."